scipy/stats/_distn_infrastructure.py

```diff
@@ -2904,6 +2919,12 @@ def fun(x, *args):
             vals = integrate.quad(fun, lb, ub, **kwds)[0] / invfac
         return vals
 
+    def _param_info(self):
+        shape_info = self._shape_info()
```

Comment:
Although the plan would be to add `_shape_info` to all distributions before the next release, we should probably add an error message here (i.e. "distribution not yet supported") in the meantime.